teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization)
+    @specialization = specialization
+    super
+  end
```

Comment:
- [ ] You are missing the other parameters as in the Person class. Kindly add them.

- [ ] Kindly ensure you pass up these parameters on the inherited **Person** class.